internal/cliwrapper/cliwrapper_interface.go

```diff
@@ -6,6 +6,7 @@ import (
 
 type CliWrapper interface {
 	ImageExists(image string) (*bool, error)
+	ContainerExists(image string) (bool, error)
```

Comment:
I can see a possible need for an "exists" check using `podman ps --all`, but I think this should be named more specifically as something like `ContainerIsRunning`.